src/app/settings/initial-settings.ts

```diff
@@ -3,11 +3,46 @@ import { defaultLanguage } from 'app/i18n';
 
 export interface Settings extends DimApiSettings {
   /** Selected columns for the Vault Organizer */
+  readonly organizerColumnsWeapons: string[];
+  readonly organizerColumnsArmor: string[];
```

Comment:
No need to put these back.

Reply:
Can do, must of missed it in the merge conflicts.